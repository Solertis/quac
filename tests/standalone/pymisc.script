#!/bin/bash

# Test some Python stuff that needs a specific external environment.
#
# Copyright (c) 2012-2013 Los Alamos National Security, LLC, and others.

. ./environment.sh

cd $DATADIR


echo '*** u.glob_maxnatural()'

x touch a b
python -c 'import u; print u.glob_maxnumeric(".")'
x touch 0
python -c 'import u; print u.glob_maxnumeric(".")'
x touch 00
python -c 'import u; print u.glob_maxnumeric(".")'
x touch 1 9
python -c 'import u; print u.glob_maxnumeric(".")'

