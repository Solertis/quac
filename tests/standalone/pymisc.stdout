*** u.glob_maxnatural()
$ touch a b
None
$ touch 0
0
$ touch 00
0
$ touch 1 9
9
